buildqtc.sh

```diff
@@ -319,6 +319,11 @@ build_unix_qtc() {
 	    make install
 	fi
 
+    if [[ $UNAME_SYSTEM == "Darwin" ]]; then
```

Comment:
indentation looks funny, otherwise lgtm

Reply:
yeah, the legacy code used mix of tabs and spaces, new code uses spaces so getting rid of the mess slowly :)